tools/ipapy2/ipapy2_clustering.py

```diff
@@ -0,0 +1,16 @@
+import click
+import pandas as pd
+from ipaPy2 import ipa
```

Comment:
if we put this small function into a `<configfile>` or use argparse we could save the click dependency and the additional container.